core/editor/src/document/document_message_handler.rs

```diff
@@ -501,10 +501,16 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 			}
 			NudgeSelectedLayers(x, y) => {
 				let paths: Vec<Vec<LayerId>> = self.selected_layers_sorted();
+
+				let delta = {
+					let neg_angle = -self.layerdata_mut(&vec![]).rotation;
+					let (cos, sin) = (neg_angle.cos(), neg_angle.sin());
+					DVec2::new(x * cos - y * sin, x * sin + y * cos)
+				};
```

Comment:
I think you can just use 
```rust
let delta = self.layerdata_mut(&vec![]).inverse((x,y).into());
```

Reply:
The `inverse` function seems to be a member of `Transform`, but I'm getting the rotation from LayerData::rotation. Do you mean I should create a transform with rotation equal to `self.layerdata_mut(&vec![]).rotation`?